compiler/daml-lf-tools/src/DA/Daml/LF/Simplifier.hs

```diff
@@ -125,6 +125,15 @@ safetyStep = \case
       BERoundNumeric      -> Safe 1
       BECastNumeric       -> Safe 0
       BEShiftNumeric      -> Safe 1
+      BEScaleBigNumeric     -> Safe 1
+      BEPrecisionBigNumeric -> Safe 1
+      BEAddBigNumeric       -> Safe 1
+      BESubBigNumeric       -> Safe 1
+      BEMulBigNumeric       -> Safe 1
+      BEDivBigNumeric       -> Safe 1
+      BEShiftBigNumeric     -> Safe 1
+      BEToNumericBigNumeric -> Safe 0
+      BEFromNumericBigNumeric -> Safe 0
```

Comment:
@sofiafaro-da or @moritzkiefer-da Could you review that when you cleaning up the compiler?

Reply:
can you add a todo please

Reply:
done.